### PR TITLE
Duralumin, Aluminum, Pewter hunger, and Glass Dagger

### PR DESCRIPTION
**Pull request by Benji123abc**
This adds the duralumin and aluminum metals to the mod, as well as pewter dragging and a glass dagger.

### CODE REVIEW

**Comment by legobmw99** [comment]
Is this based around the aluminum/duralumin that exists in the most recent versions?

**Review by legobmw99 — CHANGES_REQUESTED** [review]
Please delete .DS_Store files

**Comment by Benji123abc** [comment]
It is not currently based off of that, no.

**Comment by legobmw99** [comment]
Can you summarize your changes?

Also, there are many .DS_Store files remaining.

**Comment by Benji123abc** [comment]
Sure, I added Duralumin which boosts the other metals while being burned, but makes them run out faster. Aluminum, which clears all other metal reserves. The glass dagger, which is made with 1 stick, and 1 glass block, and has a low attack damage, but higher attack speed. And I added "pewter dragging", which, while burning pewter, fills your hunger bar. When you stop burning pewter, you then lose the hunger that you would have lost normally.

**Comment by legobmw99** [comment]
Could you break down what Duralumin does specifically for each of other metals?

**Comment by Benji123abc** [comment]
For pewter it simply increases buffs like strength, speed, and jump boost. For tin it gives blindness and nausea to mimic the effects in the books. And for iron and steel, it increases power of the pushing and pulling mechanic. It does not effect any of the other metals.

**Comment by legobmw99** [comment]
I'm unsure if you're aware, but the latest versions implement not only duralumin and aluminum, but all the other metals (and an obsidian dagger, rather than glass). I think I'd prefer new things in official earlier versions to be backports rather than divergent branches.

At the very least, I would want duralumin to be consistent cross version, including all the metals it could.
